batch/batch/front_end/front_end.py

```diff
@@ -448,7 +486,7 @@ async def _query_batches(request, user):
         where_conditions.append('(id < %s)')
         where_args.append(last_batch_id)
 
-    q = request.query.get('q', '')
+    q = request.query.get('q', f'user:{user}')
```

Comment:
This needs to be consistent with the UI. Currently the UI will show no search term. We should probably explicitly update the `request.query` term in both `get_batches` and `ui_batches`. That ensures that when we regenerate the page in `ui_batches`, we'll use the right query term.